auto_nag/tests/test_query_creator.py

```diff
@@ -1,35 +1,27 @@
 
-try:
-    # If relman-auto-nag is installed
-    from auto_nag.bugzilla.utils import os
-    from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
-                                                createQueriesList)
-    import shutil
-except:
-    # If relman-auto-nag not installed, add project root directory into
-    # PYTHONPATH
-    import os
-    import sys
-    import inspect
-    import shutil
-    currentdir = os.path.dirname(os.path.abspath(
-                                 inspect.getfile(inspect.currentframe())))
-    parentdir = os.path.dirname(currentdir)
-    sys.path.insert(0, parentdir)
-    from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
-                                                createQueriesList)
+from auto_nag.bugzilla.utils import os
+from auto_nag.bugzilla.utils import get_project_root_path
+from auto_nag.scripts.query_creator import (getTemplateValue, getReportURL,
+                                            createQueriesList, cleanUp)
+import shutil
+import datetime
 
 class TestQueryCreator:
-    def test_getTemplateValue(self):
+    def setUp(self):
+        self.queries_dir = get_project_root_path() + 'queries/'
+
+    def test_1(self):
         """
+        Tests for getTemplateValue,
         Expecting a VERSION number
         """
         url = "https://wiki.mozilla.org/Template:BETA_VERSION"
         beta_version = getTemplateValue(url)
         assert type(int(beta_version)) is not type(int)
 
-    def test_getReportURL(self):
+    def test_2(self):
```

Comment:
Sylvestre,

Nosetests will execute only alphabetically. So these methods should execute
sequentially, until and unless we give names like this, how could we
control it's execution sequentially if we give other names . I am giving
the docstring properly.

Please do let me know about your thoughts!

On Tue, Nov 3, 2015, 4:18 PM Sylvestre Ledru <notifications@github.com>
wrote:

> In auto_nag/tests/test_query_creator.py
> <https://github.com/mozilla/relman-auto-nag/pull/53#discussion_r43734624>:
>
> >          Expecting a VERSION number
> >          """
> >          url = "https://wiki.mozilla.org/Template:BETA_VERSION"
> >          beta_version = getTemplateValue(url)
> >          assert type(int(beta_version)) is not type(int)
> >
> > -    def test_getReportURL(self):
> > +    def test_2(self):
>
> Could you rename it to something more explicit?
> thanks
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/relman-auto-nag/pull/53/files#r43734624>.
>
-- 

~anoop_valluthadam